test/development/acceptance-app/rsc-build-errors.test.ts

```diff
@@ -320,66 +320,6 @@ createNextDescribe(
       await cleanup()
     })
 
-    it('should be possible to open the import trace files in your editor', async () => {
```

Comment:
Already a test for this case in `test/development/acceptance-app/editor-links.test.ts`